Automattic/portkey-android
# Stories library part 4 - media picker

This PR builds on top of #336 
Related WPAndroid PR: https://github.com/wordpress-mobile/WordPress-Android/pull/11939

In this PR we're consisting the `EXTRA_MEDIA_URI` and `EXTRA_MEDIA_URI_LIST` modes of adding media to match the current `EXTRA_MEDIA_URIS` in the WPAndroid host app, which will make requirements easier to implement.

The rest of the functionality (to provide an external media picker and be able to launch it then consume the selection from there) is already in place as part of #344, where we needed to extract things to avoid conflicts.

To test:
1. build and run the demo app
2. tap on the fab + button
3. tap on the upload icon
4. observe the picker is shown
5. select some items and then the checkmark
6. observe these are added to the Story
